modules/geo-layers/src/tile-layer/utils.js

```diff
@@ -177,11 +177,12 @@ export function getTileIndices({
   extent,
   tileSize = TILE_SIZE,
   modelMatrix,
-  modelMatrixInverse
+  modelMatrixInverse,
+  zoomOffset = 0
 }) {
   let z = viewport.isGeospatial
     ? Math.round(viewport.zoom + Math.log2(TILE_SIZE / tileSize))
-    : Math.ceil(viewport.zoom);
+    : Math.ceil(viewport.zoom) + zoomOffset;
```

Comment:
This should apply to both geospatial and non-geospatial. For example, Google Map's zoom is OSM z plus one, and although `tileSize` can be used to achieve the zoom shift, it is counter-intuitive.